nservicebus/operations/performance-counters_note_perfcounters_[1,).partial.md

```diff
@@ -0,0 +1 @@
+While the standard performance counters in the NServiceBus 6 package are marked obsolete, for compatibility reasons they are still enabled by default.
```

Comment:
so this is true for NSB v7 as well?

Reply:
You mean the naming of the file isn't correct? In NSB7 they should be completely removed. How would I name the file, if it needs to be <2.0?

Reply:
sorry my comment was incorrect